src/mlpack/core/cv/metrics/ssim_index_impl.hpp

```diff
@@ -25,7 +25,8 @@ double SSIMIndex::Evaluate(DataType const& image,
  {
   for ( int column = 0; column<11; column++)
   {
-   gaussianWindow(row, column) = gaussianKernel.Evaluate(arma::linspace(-6, row - 5, 1), arma::linspace(-6, column - 5, 1));
+   gaussianWindow(row, column) = gaussianKernel.Evaluate(arma::linspace(-6, row - 5, 1),
+                                                         more stylearma::linspace(-6, column - 5, 1));
```

Comment:
Hey, I think you accidentally type name of the commit here.

Reply:
@kartikdutt18 I am writing the comments now, and referred to the #2323, There were expressions like these `@f$R^2 =1.0@f$`. I don't know what they mean, where should I learn about them?

Reply:
I think you can use [this](https://latex2image.joeraut.com) to convert that syntax to image. You could use this [link](https://www.overleaf.com/learn/latex/mathematical_expressions) as a source of reference.